src/WebUI/Pods/PodLog.tsx

```diff
@@ -42,7 +42,24 @@ export class PodLog extends React.Component<IPodLogProps, IPodLogState> {
         );
     }
 
+    public componentWillUnmount(): void {
+        if (this._heightCssElement && document && document.head) {
+            document.head.removeChild(this._heightCssElement);
+            this._heightCssElement = undefined;
+        }
+    }
+
     public componentDidMount(): void {
+        if (!this._heightCssElement) {
+            this._heightCssElement = document.createElement("style");
+            this._heightCssElement.type = "text/css";
+            if (document && document.head) {
+                document.head.appendChild(this._heightCssElement);
+            }
+        }
+
+        this._heightCssElement.innerText = ".kubernetes-container .pod-overview-full-size { height: unset; min-height: 100%; }";

```

Comment:
in two places we are using pod-overview-full-size to set height to 100%, i think, if we set this min-height to 100% in one place may be good enough. i will check and get back to you. in that we do not need this work around.

Reply:
i have updated the PR with few changes, please check